pipelined/bevy_pbr2/src/alpha.rs

```diff
@@ -0,0 +1,22 @@
+use bevy_ecs::reflect::ReflectComponent;
+use bevy_reflect::Reflect;
+
+// FIXME: This should probably be part of bevy_render2!
```

Comment:
@cart Where do you think this should live?

Reply:
The set of available values here is specific to the `bevy_pbr2` implementation, so I think the current location is good for now.